src/pages/experiments/[experimentId]/data-exploration/components/generic-gene-table/GeneTable.jsx

```diff
@@ -208,8 +210,8 @@ const GeneTable = (props) => {
         dataSource={renderRows(data)}
         loading={loading}
         size='small'
-        pagination={{ ...tableState?.pagination, total }}
-        sorter={tableState?.sorter}
+        pagination={{ ...tableState ?.pagination, total }}
```

Comment:
find a way and fix your linter to stop correcting these, because this can cause an actual problem